### PR TITLE
Add Tornjak test

### PR DESCRIPTION
**Pull request by mrsabath**

### CODE REVIEW

File: .github/tests/tornjak/post-install.sh
Comment (marcofranssen) [inline .github/tests/tornjak/post-install.sh:1]: Probably take this one as a base.

https://github.com/spiffe/helm-charts/blob/main/.github/tests/namespace-override/post-install.sh

This got recent fixes and improvements

File: .github/tests/tornjak/pre-install.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+kubectl create namespace spire-system
+kubectl label namespace spire-system pod-security.kubernetes.io/enforce=privileged
+kubectl create namespace spire-server
+kubectl label namespace spire-server pod-security.kubernetes.io/enforce=restricted
+kubectl create namespace tornjak-fe
```
Comment (marcofranssen) [inline .github/tests/tornjak/pre-install.sh:7]: Is this split namespace setup required for tornjak. For test simplicity I wouldn't duplicate that test here.

File: examples/tornjak/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Recommended production setup
+
+Too install Spire with the least privileges possible we deploy spire across 2 namespaces.
+
+```shell
+kubectl create namespace "spire-system"
+kubectl label namespace "spire-system" pod-security.kubernetes.io/enforce=privileged
+kubectl create namespace "spire-server"
+kubectl label namespace "spire-server" pod-security.kubernetes.io/enforce=restricted
```
Comment (marcofranssen) [inline examples/tornjak/README.md:9]: I wouldn't mix tornjak with prod example.

File: charts/spire/charts/spire-server/templates/tests/test-tornjak-connection.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  namespace: {{ include "spire-server.namespace" . }}
+  labels:  
+  annotations:
+    "helm.sh/hook": test
+spec:
+  securityContext:
+    {{- toYaml .Values.podSecurityContext | nindent 4 }}
+  containers:
+    - name: wget-tornjak-backend
+      image: busybox
```
[the remark this answers is not in the document]
Comment (mrsabath) [reply charts/spire/charts/spire-server/templates/tests/test-tornjak-connection.yaml]: rebased with 'tornjak-split' branch fixed it

**Comment by mrsabath** [comment]
I will create a new PR to create the tests. Due to several re-base actions I corrupted this request to the point that is easier to create a new one.

**Pull request closed without merging**